packages/marketplace/src/components/ui/developer-analytics/detailed/detailed-tab.tsx

```diff
@@ -22,6 +21,7 @@ import { AppInstallationsState } from '@/reducers/app-installations'
 import { DeveloperState } from '@/reducers/developer'
 import { AppUsageStatsState } from '@/reducers/app-usage-stats'
 import TrafficEventTable from './traffic-event-table'
+import { prepareDefaultFilterDateParams } from './filter-bar/default-filter-group'
```

Comment:
all filterForm, filterBar, default fetch data from analytic page default should be fetch from `last 7 days` (`defaultParams`)